.github/workflows/python-publish.yml

```diff
@@ -17,15 +17,15 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: '3.7'
+          python-version: '3.12'
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install setuptools wheel twine -r requirements.txt
       - name: Build and publish
         env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_USERNAME: '__token__'
```

Comment:
not sure what this does? would we not need to keep the correct username?

Reply:
Since 2024, PyPI requires ALL of its devs to have an API token to publish new packages to the API as some sort of two-factor authentication measure. Publishing packages with a username and password has been deprecated.

Reply:
See https://pypi.org/help/#apitoken

Reply:
Anything you can do about the API token? @Esmeray6

Reply:
Noted, let me see if i can figure it out.